example/classifier/odp_classifier.c

```diff
@@ -892,6 +892,8 @@ static int parse_pmr_policy(appl_args_t *appl_args, char *optarg)
 	len = strlen(optarg);
 	len++;
 	pmr_str = malloc(len);
+	if (pmr_str == NULL)
+		return -1;
```

Comment:
An error message would be useful. E.g. `ODPH_ERR("malloc() failed\n");`. Also, the copyright year can be updated for this file.

Reply:
Log added in v2.